bindings/matrix-sdk-ffi/src/authentication_service.rs

```diff
@@ -176,10 +177,15 @@ impl HomeserverLoginDetails {
 impl AuthenticationService {
     /// Creates a new service to authenticate a user with.
     #[uniffi::constructor]
+    // TODO: This has too many arguments, even clippy agrees. Many of these methods are the same as
+    // for the `ClientBuilder`. We should let people pass in a `ClientBuilder` and possibly convert
+    // this to a builder pattern as well.
+    #[allow(clippy::too_many_arguments)]
     pub fn new(
         base_path: String,
         passphrase: Option<String>,
         user_agent: Option<String>,
+        additional_root_certificates: Vec<Vec<u8>>,
```

Comment:
In the FFI layer, could we use a type alias instead of `Vec<u8>` meaning a PEM or "der" certificate?

Reply:
Added: https://github.com/matrix-org/matrix-rust-sdk/pull/3126/commits/df1004fc181351b0ae553946df02468df53f9ca1